addons/website/static/src/less/website.edit_mode.less

```diff
@@ -44,9 +44,15 @@
 
     &.oe_structure.oe_empty, &[data-oe-type=html] {
         &:empty {
-            height: 300px;
-            background: url(/website/static/src/img/drag_here.png) center no-repeat;
-            background-size: 380px;
+            border: 2px dashed lighten(@gray-light, 20%);
```

Comment:
Do not use bootstrap variables to style the editor, this will conflict with themes. You should use a color variable defined in the `web` or `web_editor` `variables.less` files